lxterminal.yaml

```diff
@@ -0,0 +1,11 @@
+name: lxterminal
+config-opts:
+  - --enable-man 
+sources:
+  # Source0
+  - type: archive
+    url: https://github.com/lxde/lxterminal/archive/0.3.2.tar.gz
+    sha256: 03c6bdc0fcf7a2e2760e780d2499b618471b0960625d73f9b77654cd58c54ec5
+post-install:
```

Comment:
Not needed.

Reply:
> Not needed.

I consulted it with the person who actually uses this program.
Typical work with this program is not limited to using built-in features, but heavily depends on many external tools that you can access through the action menu. You have several predefined actions, but usually users define their own set.
To make sense, at least two tools must be available:
- A simple program that lets you view data from standard input in a user-friendly way, preferably a simple text editor that allows you to save data to a file. Leafpad works great for that.
- A terminal program, in which you can test commands before you put them into action menu. LXTerminal seems to be a good choice here.

Please keep in mind that in the first case you can't use xdg-open, because it requires a actual file. It is just not suitable for that.

Use case examples:
- Send image to another program, e.g. GIMP, GQview, etc.
- Take a time delayed screenshot.
- View image information using `identify`, `exif`, `file` and similar tools.
- Create thumbnails in temp directory.
- Remove spaces from filenames.
- Repair filename extension (e.g. `filename.jpg.jpg` → `filename.jpg`).
- Manipulate images from current directory using ImageMagick and Gifsicle.
- Use ImageMagick to compare images from current directory.
- Convert images from temp directory to black and white, grayscale or 256 color.
- Rescale image using ImageMagick.

For this reason, we actually need at least a terminal program, a text editor, a set of tools (ImagaMagick, Gifsicle, exif, etc.), and documentation for them.
Of course, you can use flatpak-spawn to access the desired program, but basic tools must be built-in. The same applies to manuals. Even if you have ImageMagick doc on the host, you can't expect it to be the same version as here.

Reply:
I'd argue if it's a good fit for Flathub if it requires other graphical applications than itself bundled.

Reply:
> I'd argue if it's a good fit for Flathub if it requires other graphical applications than itself bundled.

All necessary tools are included in this package. Moreover, reliance on external tools is nothing special. Actually, a lot applications require programs from the ImageMagick or FFmpeg bundle.